src/component/bookComponent/book-name.component.js

```diff
@@ -1,22 +1,21 @@
 import React from "react";
```

Comment:
I'm assuming you're using a newer version of react. "Import React from "react"" is not needed anymore